src/OrchardCore.Modules/OrchardCore.Seo/Views/SeoMetaPart.Edit.cshtml

```diff
@@ -0,0 +1,154 @@
+@model OrchardCore.Seo.ViewModels.SeoMetaPartViewModel
+
+@await DisplayAsync(await New.ShortcodeModal())
+
+<div class="form-group">
+    <div class="custom-control custom-checkbox">
+        <input type="checkbox" class="custom-control-input" asp-for="Render" checked="@Model.Render" />
+        <label class="custom-control-label" asp-for="Render">@T["Render"]</label>
+        <span class="hint">— @T["Whether to render metadata for this page."]</span>   
+    </div> 
+</div>
+
+<div class="form-group" asp-validation-class-for="PageTitle">
+    <label asp-for="PageTitle">@T["Page Title"] </label>
+    <input asp-for="PageTitle" class="form-control content-preview-text" />
+    <span class="hint">@T["Override the default generated web browser page title (recommended to keep under 70 characters)."]</span>
+</div>  
+
+<div class="form-group" asp-validation-class-for="MetaDescription">
+    <label asp-for="MetaDescription">@T["Meta Description"] </label>
+    <textarea asp-for="MetaDescription" class="form-control content-preview-text" rows="5"></textarea>
+    <span class="hint">@T["The meta description to render for this page."]</span>
+</div>
+
+<style asp-name="codemirror"></style>
+<script asp-name="codemirror" depends-on="admin" at="Foot"></script>
+<script asp-name="codemirror-addon-display-autorefresh" at="Foot"></script>
+<script asp-name="codemirror-addon-selection-active-line" at="Foot"></script>
+<style asp-name="codemirror"></style>
+<script at="Foot">
+    $(function () {
+        var optionsTextArea = document.getElementById('@Html.IdFor(x => x.MetaDescription)');
+        if (optionsTextArea) {
+            var editor = CodeMirror.fromTextArea(optionsTextArea, {
+                autoRefresh: true,
+                lineWrapping: true,
+                styleActiveLine: true
+            });
+            
+            initializeCodeMirrorShortcodeWrapper(editor);
```

Comment:
I thought you dropped codemirror for this field? and kept if for the google schema? so this would just be a text area, and the google schema would have the shortcodes?

Reply:
Does `initializeCodeMirrorShortcodeWrapper `work with just a textarea?

Reply:
see the html content field multiline.

I think this should work
```
@await DisplayAsync(await New.ShortcodeModal())

<div class="form-group">
    <label asp-for="Html">@Model.PartFieldDefinition.DisplayName()</label>
    <textarea asp-for="Html" rows="5" class="form-control content-preview-text shortcode-modal-input" dir="@culture.GetLanguageDirection()"></textarea>
    @if (!String.IsNullOrEmpty(settings.Hint))
    {
        <span class="hint">@settings.Hint</span>
    }
</div>

Reply:
ok, `shortcode-modal-input` worked. Added on multiple inputs.